services/frontend-service/src/ui/components/TopAppBar/TopAppBar.tsx

```diff
@@ -145,18 +135,11 @@ export const TopAppBar: React.FC<TopAppBarProps> = (props) => {
                 {renderedUser}
                 <div className="mdc-top-app-bar__section mdc-top-app-bar__section--align-end">
                     <strong className="sub-headline1">Planned Actions</strong>
-                    <Button
-                        className="mdc-show-button mdc-button--unelevated"
-                        icon={<ShowBarWhite />}
-                        onClick={toggleSideBar}
-                        highlightEffect={false}
-                    />
                     <SideBar
-                        className={classNames(`mdc-drawer-sidebar mdc-drawer-sidebar-container`, {
-                            'mdc-drawer-sidebar--displayed': sideBar,
-                            'mdc-drawer-sidebar--hidden': !sideBar,
-                        })}
-                        toggleSidebar={toggleSideBar}
+                        className={classNames(
```

Comment:
The sidebar overlaps the user welcome message, as shown.
![Screen Shot 2024-08-20 at 2 02 39 PM](https://github.com/user-attachments/assets/e66caf66-7759-4ab2-abc5-6f2c858cb351)

Reply:
should look a lot better now:
![image](https://github.com/user-attachments/assets/d27c8016-28f7-448e-81cf-b5acf4a8451a)